File: opentsdb-proxy/src/main/java/net/opentsdb/proxy/netty/rpc/VersionRpc.java
package net.opentsdb.proxy.netty.rpc;

import org.jboss.netty.channel.Channel;

public class VersionRpc implements TelnetRpc {
  @Override
  public void execute(Channel chan, String[] command) {
    if (chan.isConnected()) {
      chan.write("OpenTSDB To Kafka Proxy " + "\n" + "Version 1.0" + "\n");
    }
  }
}

File: opentsdb-proxy/src/main/scala/net/opentsdb/proxy/netty/rpc/VersionRpc.scala
package net.opentsdb.proxy.netty.rpc

import org.jboss.netty.channel.Channel

class VersionRpc extends TelnetRpc{
  def execute(chan: Channel, command: Array[String]) {
    if (chan.isConnected) {
      chan.write("OpenTSDB To Kafka Proxy " + "\n" + "Version 1.0" + "\n");
    }
  }
}
